packages/functional-tests/pages/resetPassword.ts

```diff
@@ -17,15 +21,25 @@ export const selectors = {
 };
 
 export class ResetPasswordPage extends BaseLayout {
+  public react = false;
```

Comment:
A bit torn if it makes sense to extend the page and overwrite functions needed vs updating the page themselves.

Reply:
Agreed that would better. I've filed a follow up, [FXA-8015](https://mozilla-hub.atlassian.net/browse/FXA-8015). I'd change it here but don't want to derail this PR.